contracts/VaultV3.vy

```diff
@@ -606,24 +578,33 @@ def _process_report(strategy: address) -> (uint256, uint256):
     else:
         loss = current_debt - total_assets
 
+    # We compute unlocked_profit
+    profit_end_date: uint256 = self.profit_end_date
+    unlocked_profit: uint256 = 0 
+    if self.profit_distribution_rate_ != 0:
```

Comment:
OPTIMISATION: 
we are using `profit_distribution_rate_` three times so better to cache it just before this line

Reply:
with the optimization done on the comment before, it makes even more sense